scripts/release.js

```diff
@@ -156,6 +156,15 @@ logger.warn(`
 
 shouldContinuePrompt();
 
+// Creates a new React test app, install the component app as dependency and build the test app to ensure no errors or build issues
```

Comment:
Would be helpful to have this as a separate PR since it is different from housekeeping one.